src/api/pulls.rs

```diff
@@ -194,6 +195,49 @@ impl<'octo> PullRequestHandler<'octo> {
     pub fn list(&self) -> list::ListPullRequestsBuilder {
         list::ListPullRequestsBuilder::new(self)
     }
+
+    /// Lists all of the `Review`s associated with the pull request.
+    /// ```no_run
+    /// # async fn run() -> octocrab::Result<()> {
+    /// let reviews = octocrab::instance().pulls("owner", "repo").list_reviews(101).await?;
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub async fn list_reviews(&self, pr: u64) -> crate::Result<Vec<crate::models::pulls::Review>> {
```

Comment:
Shouldn't this return a `Page` and not just `Vec`? Presumably if it's PR with 100+ reviews this wouldn't give you all of them.

Reply:
Right, oops!